rever/logger.xsh

```diff
@@ -38,6 +38,8 @@ class Logger:
             entry['activity'] = activity
         if data is not None:
             entry['data'] = data
+        if version is not None:
+            entry['version'] = version
```

Comment:
It would be nice if version is None, that this sets it to $VERSION`, so we don't always have to pass it in.

Reply:
Are we always guaranteed a $VERSION? (Eg setup runs)

Reply:
Yes it will always be there